lib/cmd/common/build.js

```diff
@@ -198,42 +198,55 @@ function mutateGitRefArgs(args) {
 function doBuild(args, cb) {
   var uri = "box/srv/1.1/wid/" + fhc.curTarget + "/" + args.destination + "/" + args.app + "/deliver";
   var doCall = function() {
-    var payload = argsToPayload(args);
-    common.doApiCall(fhreq.getFeedHenryUrl(), uri, payload, i18n._("Error reading app: "), function(err, data) {
-      if (err) {
-        return cb(err);
-      }
-      var keys = [];
-      if (data.cacheKey) keys.push(data.cacheKey);
-      //      if(data.stageKey) keys.push(data.stageKey);
-      if (keys.length > 0) {
-        async.map(keys, common.waitFor, function(err, results) {
+
+    //Fix missing creds names - RHMAP-9456
+    if (args.bundleId) {
+      common.findCredentialsByID(args.bundleId, function(err, data) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)